api/src/main/resources/emails/tier_access_threshold/content.html

```diff
@@ -163,7 +164,7 @@
                   <tbody>
                     <tr>
                       <td align="left" style="font-size:0px;padding:10px 25px;word-break:break-word;">
-                        <div style="font-family:'Montserrat', sans-serif;font-size:24;font-weight:600;line-height:1.5;text-align:left;color:#000000;">Hello ${FIRST_NAME},</div>
+                        <div style="font-family:'Montserrat', sans-serif;font-size:16px;font-weight:600;line-height:1.5;text-align:left;color:#000000;">Hello ${FIRST_NAME},</div>
```

Comment:
my version of mjml said "24" was invalid as a size; px was required.

I updated it to 16px to match the other email.